src/main/org/nlogo/prim/_linkbreedvariableof.scala

```diff
@@ -0,0 +1,41 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.prim
+
+import org.nlogo.agent.{ Agent, AgentSet }
+import org.nlogo.api.{ AgentException, I18N, LogoException, LogoListBuilder, Syntax }
+import org.nlogo.nvm.{ ArgumentTypeException, Context, EngineException, Reporter }
+
+class _linkbreedvariableof(name: String) extends Reporter {
+  override def syntax: Syntax =
+    Syntax.reporterSyntax(
+      Array[Int](Syntax.LinksetType | Syntax.LinkType),
+      Syntax.WildcardType)
+
+  override def toString: String = s"${super.toString}:$name"
+
+  override def report(context: Context): AnyRef = args(0).report(context) match {
+      case agent: Agent =>
+        if (agent.id == -1)
+          throw new EngineException(context, this,
+            I18N.errors.getN("org.nlogo.$common.thatAgentIsDead", agent.classDisplayName))
+        try {
```

Comment:
I did this because the `catch` has curly braces, and it looked sot of awkward to have the `try` without and the `catch` with.